docs/content/python/getting-started/quickstart/index.md

```diff
@@ -87,59 +85,29 @@ resource(_type: Page, "page", actions, roles) if
             permissions: ["read"]
         },
         admin: {
-            permissions: ["write"],
-            implies: ["guest"]
+            permissions: ["write", "read"]
         }
     };
  ```
 
-The `allow` rule is the top-level rule that we use to say who can do what in our application.
-In this case, we are delegating to Oso's built-in `role_allows` rule which implements all the
-authorization logic for role-based access control based on the data we provide in `actor_has_role_for_resource`
-and `resource`.
-
-An `actor_has_role_for_resource` rule looks up role objects that are associated with an actor.
-Role objects are of the form `{name: "the-role-name", resource: TheResourceObject}`.
-The Oso builtin roles will look up this rule, so this is required.
+The `allow` rule is what's queried by `is_allowed()` in your application.
+In this case, we are delegating to Oso's built-in `role_allows` rule.
+`role_allows` will use the data we provide in `actor_has_role_for_resource`.
 
-A `resource` rule governs access to a specific resource in your app — for instance, a page, an object, a route, or a database entry.
+A `resource` rule governs access to a specific resource in your app.
 In the above configuration for `Page`, there are two possible actions: `"read"` and `"write"`.
 Guests can read, but not write, to a `Page`.
 Admins are allowed to write to a `Page`.
 
-The line `implies: ["guest"]` says that admins can also do anything guests are allowed to do; that is, admins can also read a `Page`.
-
 There's much more you can do with Oso — we're just scratching the surface here. Polar is a very expressive language, and you can encode authorization logic far beyond what we've shown in this guide.
 
-## Calling back into your {{% lang %}} code
-
-You can call properties and methods on your {{% exampleGet "objects" %}} from Polar.
-These will defer control back to your app.
-Oso leaves the decision of how to store role assignments up to you — you might choose to store those role assignments in a database, in memory, or create them dynamically.
-Our `actor_has_role_for_resource` rule calls the {{% exampleGet "methods" %}} `{{% exampleGet "getroles" %}}` to get all the roles for our actor.
-
-```polar
-actor_has_role_for_resource(actor, role_name, resource) if
-    role in actor.get_roles() and
-    role_name = role.name and
-    resource = role.resource;
- ```
-
-To refer to your {{% exampleGet "classes" %}} in Polar, you must _register_ them with Oso.
-
-{{% exampleGet "register_classes" %}}
-
 ## Want to talk it through?
 
 If you have any questions, are getting stuck, or just want to talk something
 through, jump into [Slack](https://join-slack.osohq.com/) and an engineer from
 the core team (or one of the hundreds of developers in the growing community)
 will help you out.
 
-## Complete Running Example
```

Comment:
We should update the example once we're good with the guide.

Did you test this updated code?